src/tree.ts

```diff
@@ -325,4 +310,51 @@ export class Tree {
   getProof(input: ProofInput): Proof {
     return createProof(this.rootNode, input);
   }
+
+  private getParentNodes(bitstring: GindexBitstring, expand = false): Node[] {
```

Comment:
Can you add a nice JSDoc comment like in setHashObjectFn explaining this function?